ng2-material/components/sidenav/sidenav.ts

```diff
@@ -210,12 +210,16 @@ export class MdSidenavContainer implements OnDestroy, AfterViewInit {
     this.children.toArray().forEach((m: MdSidenav) => {
       m.backdropRef = this._backdrop;
     });
+    
+    /*
```

Comment:
If the code is not required anymore, you have to remove it. Versioning allow us to fallback if the solution was bad